action.yml

```diff
@@ -11,7 +11,7 @@ inputs:
   sceptre_version:
     description: "Version of Sceptre to install"
     required: false
-    default: "2.5.0"
+    default: ""
```

Comment:
will the action run with the latest version of sceptre if user does not specify a `sceptre_version`?  or is this really a required setting?  if required i think it will break current functionality?